### PR TITLE
ArraySink.a -> ArraySink.array

### LINKED ISSUE #251: ArrayDAO and ArraySink should have matching property names
It's annoying to remember which one has `.array` and which one has `.a`.

### PR DESCRIPTION
**Pull request by mdittmer**
Fixes #251

### CODE REVIEW

File: src/com/google/foam/demos/clock/Clock.js [hunk 1]
```diff
@@ -75,9 +75,9 @@ foam.CLASS({
 
       var date = new Date();
 
-      this.secondHand.a = Math.PI/2 - Math.PI*2 * date.getSeconds() / 60 ;
-      this.minuteHand.a = Math.PI/2 - Math.PI*2 * date.getMinutes() / 60 ;
-      this.hourHand.a   = Math.PI/2 - Math.PI*2 * (date.getHours() % 12) / 12 + this.minuteHand.a / 12;
+      this.secondHand.array = Math.PI/2 - Math.PI*2 * date.getSeconds() / 60 ;
+      this.minuteHand.array = Math.PI/2 - Math.PI*2 * date.getMinutes() / 60 ;
+      this.hourHand.array   = Math.PI/2 - Math.PI*2 * (date.getHours() % 12) / 12 + this.minuteHand.array / 12;
```
Comment (jacksonic) [inline src/com/google/foam/demos/clock/Clock.js]: Was this 'a' meant to be 'angle'?

File: src/com/google/foam/demos/lifestar/LifeStar.js [hunk 2]
```diff
@@ -49,7 +49,7 @@ foam.CLASS({
           this.y *= s;
           var t = this.transform;
           t.scale(s, s);
-          x.transform(t.a, t.d, t.b, t.e, t.c, t.f);
+          x.transform(t.array, t.d, t.b, t.e, t.c, t.f);
```
Comment (jacksonic) [inline src/com/google/foam/demos/lifestar/LifeStar.js]: Another overzealous replacement

File: src/com/google/foam/demos/threedtest/ThreeDTest.js [hunk 3]
```diff
@@ -42,7 +42,7 @@ foam.CLASS({
           this.y *= s;
           var t = this.transform;
           t.scale(s, s);
-          x.transform(t.a, t.d, t.b, t.e, t.c, t.f);
+          x.transform(t.array, t.d, t.b, t.e, t.c, t.f);
```
Comment (jacksonic) [inline src/com/google/foam/demos/threedtest/ThreeDTest.js]: Should be 'a'

File: src/com/google/net/OAuth2.js [hunk 4]
```diff
@@ -88,14 +88,14 @@ foam.CLASS({
...
 
         if ( ! auth2.isSignedIn.get() ) {
           // kick off GAPI_SIGNIN_PROMISE if non-existent
-          if ( ! self.GAPI_SIGNIN_PROMISE.a ) {
-            self.GAPI_SIGNIN_PROMISE.a = auth2.signIn();
+          if ( ! self.GAPI_SIGNIN_PROMISE.array ) {
+            self.GAPI_SIGNIN_PROMISE.array = auth2.signIn();
           }
-          return self.GAPI_SIGNIN_PROMISE.a.then(self.onSignIn.bind(self)).then(SUPER).then(self.completeSend);
+          return self.GAPI_SIGNIN_PROMISE.array.then(self.onSignIn.bind(self)).then(SUPER).then(self.completeSend);
```
Comment (jacksonic) [inline src/com/google/net/OAuth2.js]: Should be 'a'?

File: src/foam/core/InnerClass.js [hunk 5]
```diff
@@ -34,7 +34,7 @@
       function init() {
         var ic1 = this.InnerClass1.create({a:1, b:2});
         var ic2 = this.InnerClass2.create({x:5, y:10});
-        log(ic1.a, ic1.b, ic2.x, ic2.y);
+        log(ic1.array, ic1.b, ic2.x, ic2.y);
```
Comment (jacksonic) [inline src/foam/core/InnerClass.js]: Should be 'a'

**Review by jacksonic — CHANGES_REQUESTED** [review]
Unintended 'a' property changes noted with comments. Change 'em back!

File: src/com/google/foam/demos/clock/Clock.js (hunk 1 above)
Comment (mdittmer) [reply src/com/google/foam/demos/clock/Clock.js]: Looks like, yes. Fixed and renamed to `angle` (pending push).

File: src/com/google/foam/demos/lifestar/LifeStar.js (hunk 2 above)
Comment (mdittmer) [reply src/com/google/foam/demos/lifestar/LifeStar.js]: Done (pending push).

File: src/com/google/foam/demos/threedtest/ThreeDTest.js (hunk 3 above)
Comment (mdittmer) [reply src/com/google/foam/demos/threedtest/ThreeDTest.js]: Done (pending push).

File: src/com/google/net/OAuth2.js (hunk 4 above)
Comment (mdittmer) [reply src/com/google/net/OAuth2.js]: Done (pending push).

File: src/foam/core/InnerClass.js (hunk 5 above)
Comment (mdittmer) [reply src/foam/core/InnerClass.js]: Done (pending push).

**Review by jacksonic — APPROVED** [review]

**Pull request merged**